kmcd-props.md

```diff
@@ -305,6 +305,7 @@ The Debt growth should be bounded in principle by the interest rates available i
     rule derive( totalDebtBoundedRun(... debt: DEBT, dsr: _  ) #as PREV , Measure(... debt: DEBT')            ) => Violated(PREV) requires DEBT' >Rat DEBT
     rule derive( totalDebtBoundedRun(... debt: DEBT, dsr: DSR)          , TimeStep(TIME, _)                   ) => totalDebtBoundedRun(... debt: DEBT +Rat (vatDaiForUser(Pot) *Rat ((DSR ^Rat TIME) -Rat 1)), dsr: DSR)
     rule derive( totalDebtBoundedRun(... debt: DEBT, dsr: DSR)          , LogNote(_ , Vat . frob _ _ _ _ _ _) ) => totalDebtBounded(... dsr: DSR)
+    rule derive( totalDebtBoundedRun(... debt: DEBT, dsr: DSR)          , LogNote(_ , Vat . suck _ _ _      ) ) => totalDebtBounded(... dsr: DSR)
```

Comment:
An alternative:

```suggestion
    rule derive( totalDebtBoundedRun(... debt: DEBT, dsr: DSR)          , LogNote(_ , Vat . suck _ _ WAD    ) ) => totalDebtBoundedRun(... debt: DEBT +Rat WAD, dsr: DSR)
```

Both approaches should largely work, but going back to `totalDebtBounded` state basically loses any temporal information we've accumulated. Up to you.

Reply:
Sounds good, might as well be as precise as possible.